src/nvim/ops.h

```diff
@@ -80,7 +80,6 @@ enum {
   OP_LOWER        = 12,  ///< "gu" make lower case operator
   OP_JOIN         = 13,  ///< "J"  join operator, only for Visual mode
   OP_JOIN_NS      = 14,  ///< "gJ"  join operator, only for Visual mode
-  OP_ROT13        = 15,  ///< "g?" rot-13 encoding
```

Comment:
should we keep a commented-out line here, so that it's obvious to others that a "slot" is available here?